NetCord/JsonModels/JsonMessagePollAnswerCount.cs

```diff
@@ -0,0 +1,12 @@
+using System.Text.Json.Serialization;
+
+namespace NetCord.JsonModels;
+
+public class JsonMessagePollAnswerCount : JsonEntity
```

Comment:
The id for the object is documented as an integer, so it shouldn't be a JsonEntity and the `Id` field should be an int I think. The same rule applies to other objects too.